### PR TITLE
Clarify Domain Verification

### PR DESCRIPTION
**Pull request by jonathansampson**
Previous wording was terribly confusing; I spent a bit of time in the Search Console looking for a place to associate a domain name with an extension.

Changes proposed in this pull request:

- Slight rewording of _Verification_ explanation.

### CODE REVIEW

File: site/en/docs/extensions/mv3/settings_override/index.md [hunk 1]
```diff
@@ -14,11 +14,7 @@ later.
 
 ## Homepage, search provider, and startup pages {: #others }
 
-Here is an example how [homepage][1], [search provider][2] and [startup pages][3] can be modified in
-the [extension manifest][4]. Web sites used in the settings API must be [verified][5] (via Webmaster
-Tools) as being associated with that item in the Chrome Web Store. Note that if you verify ownership
-for a domain (for example, https://example.com) you can use any subdomain or page (for example,
-https://app.example.com or https://example.com/page.html) within your extension.
+Here is an example how [homepage][1], [search provider][2] and [startup pages][3] can be modified in the [extension manifest][4]. Any domain used in the settings API must be [verified][5] (via Google Search Console) by the same developer account publishing the extension. Note that if you verify ownership for a domain (for example, https://example.com) you can use any subdomain or page (for example, https://app.example.com or https://example.com/page.html) within your extension.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/settings_override/index.md:17]: Rewrap at 100 characters.

```suggestion
Here is an example how [homepage][1], [search provider][2] and [startup pages][3] can be modified in
the [extension manifest][4]. Any domain used in the settings API must be [verified][5] (via Google
Search Console) by the same developer account publishing the extension. Note that if you verify
ownership for a domain (for example, https://example.com) you can use any subdomain or page (for
example, https://app.example.com or https://example.com/page.html) within your extension.
```

**Review by dotproto — CHANGES_REQUESTED** [review]
The content update looks good. We just need to wrap the text at 100 characters before merging.

**Comment by jonathansampson** [comment]
Thank you, @dotproto. I went ahead and wrapped _on spaces_ at 100 characters, merged with previous commits, and force pushed the changes for further review. Kind regards!

**Review by dotproto — APPROVED** [review]
LGTM. Thanks, @jonathansampson 😄

**Pull request merged**